### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Adorfer**
typos in sensor definitions. 
"PM2.5" was as name in PM10, plus probably non working matchings in 
/lib/decoding/luftdatenHandler.js
(i am still not sure how this this working, i added a real example from the debug console to the file in the comments for verification)
grafik

### CODE REVIEW

File: lib/decoding/luftdatenHandler.js [hunk 1]
```diff
@@ -38,18 +38,34 @@ const { transformAndValidateMeasurements } = require('./validators'),
...
  * [
  *   {"sensor_id": "sensorID", "value":"value" },
  *   ...
  * ]
+ *
  */
 
 const matchings = {
-  p0: ['pm01', 'pm1.0'],
+  p0: ['pm01', 'pm0', 'p1.0', 'p0'],
```
Comment (ubergesundheit) [inline lib/decoding/luftdatenHandler.js]: This object is used to match the json keys from the luftdaten json to the `title` of openSenseMap sensor definitons.
So if you define a sensor with the title `pm1.0`, the value should appear in the `p0` array. 
So the values as they were seem good enough for me.

Also, your changes directly below are not correct I think.

**Comment by ubergesundheit** [comment]
I suggest you add a test for the device to https://github.com/Adorfer/openSenseMap-API/blob/development/tests/tests/010-luftdaten-test.js

You can then run the tests with docker and docker-compose installed if you just execute `run-tests.sh` in the main directory. I don't know if it works on windows.

**Comment by Adorfer** [comment]
> I don't know if it works on windows.

Sorry, i have no idea what the goal of that is.

**Comment by ubergesundheit** [comment]
The goal adding a test is to make sure your changes are working as intended and for future changes not to break the code you've written.

**Comment by Adorfer** [comment]
the point which i did not get was the windows-reference. Does the evaluation of the sensor need a windows-Environment? What is that like? WindowsServer? Desktop-Windows10?
It sounds very strange that it's not possible in a standard debian/ubuntu/arch environment.

**Comment by ubergesundheit** [comment]
Sorry, I did not know which Os you are using so I just wanted to say that the run-tests.sh most likely does not work on windows. (I’ve never tested so far)

If you want to add the tests, I would be really happy!

**Pull request merged**